simple_template_server/README.md

```diff
@@ -0,0 +1,50 @@
+Template Development Server
+===================================
+
+This is a simple Snippet Server for development of Snippet Templates. It ignores
+the client configuration including locale, Firefox version and release channels
+and just returns all the snippets to the requesting client. If you only care for
+Snippet Template development this is the server for you.
+
+Installation
+------------
+
+1. Create and activate a Python virtualenv
+
+    ``` shell
+    virtualenv venv
+    . venv/bin/activate
+    ```
+
+2. Install the needed packages
+
+    ``` shell
+    pip install -r requirements.txt
```

Comment:
It gets pretty far now, but now it's hitting:

```
Requirement already satisfied (use --upgrade to upgrade): setuptools>=1.0 in ./venv/lib/python2.7/site-packages (from cryptography==1.2.3->-r requirements.txt (line 140))
THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    msgpack-python==0.4.7 from https://pypi.python.org/packages/f8/db/20f9f719ca0fbd2a8934d672a3a198e992e77039ac019cd1cd829e405d2c/msgpack_python-0.4.7-cp27-cp27mu-manylinux1_x86_64.whl#md5=a9e560d5acc1bd70b9a32aa24b96433a (from -r requirements.txt (line 131)):
        Expected sha256 5e001229a54180a02dcdd59db23c9978351af55b1290c27bc549e381f43acd6b
             Got        cc38b1e90b9f5ddc0ffee573bb686268ed95f68bca0e8cc953e32fdda98993bd

```

Doesn't seem like an obvious "install some dev dependency" so not sure how to fix, but might be related to the `pip install --upgrade pip` thing I need to do?